evalml/data_checks/class_imbalance_data_check.py

```diff
@@ -146,7 +147,15 @@ def validate(self, X, y):
         """
         messages = []
 
+        original_vc = pd.Series(y).value_counts(sort=True)
         y = infer_feature_types(y)
+        new_vc = y.value_counts(sort=True)
+        reverse_mapping = {
```

Comment:
Might be nice to name this and `normal_mapping` to something more easily understandable, like `ww_transformed_mapping` or something of the sort. Would make comprehending this code much easier.